SparkPost/matchbox
# UX-486 Resolve certain security vulnerabilities

### What Changed

Upgrades the following for `ssri`:
- libby-react 
- lerna
- webpack

Uninstalls Playroom (No webpack 5 support)

See: https://github.com/SparkPost/matchbox/security/dependabot/package-lock.json/ssri/open
Note: lerna-changelog hasn't released a fix and the alert will not be dismissed

---

Upgrades the following for `ssir` `is-svg` `react-dev-utils`:
- Gatsby

See: https://github.com/SparkPost/matchbox/security/dependabot/site/package-lock.json/is-svg/open
https://github.com/SparkPost/matchbox/security/dependabot/site/package-lock.json/ssri/open
https://github.com/SparkPost/matchbox/security/dependabot/site/package-lock.json/react-dev-utils/open

Note: This should dismiss 3 alerts

### How To Test or Verify
- Verify libby runs and builds correctly
- Verify the site runs and builds correctly

### PR Checklist
